tests/morgan-goodbye-present.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
```

Comment:
Should we turn this file into a real test?!

Reply:
good catch! i think i'll delete it, the error was captured by relu tests anyway